src/content/docs/developer/modules/ui-modules/custom-ai.mdx

```diff
@@ -112,7 +112,9 @@ For Crowdin Enterprise:
 
 ## Communication between Custom AI App and Crowdin
 
-The system sends prompts to the app using `chatCompletionsUrl`, the app then processes the prompts and responds to the system with a reply. When configuring a Custom AI provider in the **AI > Providers** page, Crowdin sends a request to the app using `modelsUrl`. Particularly to display available models in the respective input field in the Custom AI provider settings page. A user can select and save needed models, which then will be used for content pre-translation and communication with the assistant using `chatCompletionsUrl`.
+Crowdin sends prompts to the app using `chatCompletionsUrl`, and the app then processes these prompts and responds to Crowdin with a reply. To enable your Custom AI as an assistant in the Crowdin Editor, the app must implement the `stream` parameter using [server-sent events](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events#event_stream_format) for real-time interaction via streaming responses. When streaming is supported, it allows AI-generated content to be delivered progressively, providing real-time feedback to users. This improves the workflow by offering immediate suggestions or corrections without waiting for a complete response, ensuring a smooth, interactive experience for tasks like confirming terminology, checking consistency, or refining translations. Crowdin includes the `stream` parameter set to `true` in requests to the app for `chatCompletions` and expects a response in [the corresponding format](#expected-response-from-the-app-streaming).
```

Comment:
> To enable your Custom AI as an assistant in the Crowdin Editor, the app must implement the `stream` parameter

Isn't this optional? The app could work in the no-streaming mode by providing the "Expected Response from the App" response.

Reply:
Updated texts according to the optionality of the `stream` parameter